pkg/driver/mount.go

```diff
@@ -272,6 +259,23 @@ func (m *S3Mounter) Mount(bucketName string, target string,
 	return nil
 }
 
+func addOptionToEnvironmentVariables(optionName string, envName string, options []string, env []string) ([]string, []string) {
```

Comment:
We should add some doc for this function: what's in the comment below is a start, plus what it is returning.

Reply:
nit: `addOptionToEnvironmentVariables` or rather something with `move...` or `convert..`?

`moveOptionToEnvironment`?

Reply:
nit: return order, should it be consistent with the parameters? so `options, env`?